<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FreqItemset</class>
 <widget class="QWidget" name="FreqItemset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Freq. Itemsets</string>
  </property>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Information</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>22</y>
      <width>201</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>From Date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>To Date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblMinSupportFilter_2">
         <property name="text">
          <string>Min. Support (%):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblMinSupportFilter_3">
         <property name="text">
          <string>Min. Confidence (%):</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="lblFomDateFilter">
         <property name="text">
          <string>12</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblToDateFilter">
         <property name="text">
          <string>12</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblMinSupportFilter">
         <property name="text">
          <string>12</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblMinConfFilter">
         <property name="text">
          <string>12</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Filter Itemsets</string>
   </property>
   <widget class="QPushButton" name="btnFilterItemset">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Contains:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtFilterContains"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="chkShow">
       <property name="text">
        <string>Show Only Support &gt;= Min. Support</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableWidget" name="itemsetTableWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>691</width>
     <height>601</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Itemsets</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Transactions</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Support Count</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Support (%)</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
